features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml

```diff
@@ -22,6 +22,11 @@
   -->
 
 <configurations>
+    <configuration type="smsaccountconfirmation" display="smsaccountconfirmation" locale="en_US" emailContentType="text/html">
```

Comment:
Can we avoid this attribute 'emailContentType'?
And if not possible change the content type to text/plain

Reply:
The attribute is referred to by the management console. Therefore, I think we cannot remove it.
+1 for changing the content type.

Reply:
Fixed : 5a20b3e2f24c3721024b8a9d5762fc385b46fa1c